scripts/apps/search/directives/SearchResults.js

```diff
@@ -253,6 +242,11 @@ export function SearchResults(
                     if (originalQuery) {
                         criteria.source.query = originalQuery;
                     }
+
+                    // finally, if it's a forced refresh then scroll to Top
```

Comment:
This comment says "if it's a forced refresh scroll to top", but the assignment `scope.scrollTop = 0` does not cause a change in scroll position.